src/screens/DecodeTx.js

```diff
@@ -5,11 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useState } from 'react';
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂